modules/network-observability-flowcollector-about.adoc

```diff
@@ -0,0 +1,121 @@
+      logLevel: info
+      privileged: false
+      resources:
+        requests:
+          memory: 50Mi
+          cpu: 100m
+        limits:
+          memory: 800Mi
+      kafkaBatchSize: 10485760
+  processor:
+    logLevel: info
+    resources:
+      requests:
+        memory: 100Mi
+        cpu: 100m
+      limits:
+        memory: 800Mi
+  loki:               <3>
+    url: 'http://loki.netobserv.svc:3100/'
+    statusUrl: 'https://loki-query-frontend-http.netobserv.svc:3100/'
+    authtoken: HOST
+    tls:
+      enable: false
+  consolePlugin:
+    register: true
+    logLevel: info
+    portNaming:
+      enable: true
+      portNames:
+        "3100": loki
+    quickFilters:    <4>
+    - name: Applications
+      filter:
+        src_namespace!: 'openshift-,netobserv'
+        dst_namespace!: 'openshift-,netobserv'
+      default: true
+    - name: Infrastructure
+      filter:
+        src_namespace: 'openshift-,netobserv'
+        dst_namespace: 'openshift-,netobserv'
+    - name: Pods network
+      filter:
+        src_kind: 'Pod'
+        dst_kind: 'Pod'
+      default: true
+    - name: Services network
+      filter:
+        dst_kind: 'Service'
+----
+<1> The Agent specification, `spec.agent.type`, must be `EBPF`. eBPF is the only {product-title} supported option.
+<2> You can set the Sampling specification, `spec.agent.ebpf.sampling`, to manage resources. Continuously sampled flow collection might consume a large amount of resources. You can mitigate this by setting a sampling ratio. A value of 100 means one flow every 100 is sampled. A value of 1 means all flows are sampled. The lower the value, the increase in returned flows and the accuracy of derived metrics. By default, eBPF sampling is set to a value of 50, for example 1:50. Note that more sampled flows also means more storage needed. It is recommend to start with default values and refine empirically, to determine which setting your cluster can manage.
+<3> The Loki specification, `spec.loki` configures how to reach Loki. The default values match the Loki install paths mentioned in the Installing the Loki Operator section, but you may have to configure differently if you used another installation method.
+<4> The `spec.quickFilters` specification defines filters that show up in the web console. The `Application` filter keys,`src_namespace` and `dst_namespace`, are negated `!`, so the `Application` filter shows shows all traffic that _do not_ originate from, nor destinate to, any `openshift-` or `netobserv` namespaces.
+
+= Flow Collector Configuration Parameters
+The fields for the Flow Collector custom resource are described in the following table.
+
+.FlowCollector custom resource configuration parameters
+[cols="1,1,8a",options="header"]
+|===
+
+|Field
+|Type
+|Description
+
+|`spec.agent.type`
+|`string`
+|Specifies the agent for flows extraction. The default value is `EBPF`, as it is the only {product-title} supported agent.
+
+|`spec.agent.ebpf`
+|`integer`
```

Comment:
This is not an `integer`, not sure what is the usual wording in doc for a structured object, maybe just "Object" ?

Reply:
Addressed, I see this now in the API. Not sure where I got integer from but may be a copy/paste holdover. I will go through this table again and make sure there's no more holdovers.